platform/openide.explorer/src/org/openide/explorer/view/OutlineView.java

```diff
@@ -876,6 +880,9 @@ final void synchronizeSelectedNodes(boolean scroll, Node... nodes) {
                 }
             }
         }
+        // [NETBEANS-4857]: re-activate property change events
+        outline.getSelectionModel().addListSelectionListener(managerListener);
```

Comment:
But shouldn't there be at least one event processed in the end?

Might it be better to do outline.getSelectionModel().setValueIsAdjusting(true) at the beginning and setValueIsAdjusting(false) at the end? Does this work?

Reply:
The ExplorerManager.PROP_SELECTED_NODES property change event is already fired before _synchronizeSelectedNodes(..)_ method is called. So i think there is no need for any more property change event within the synchronize method.

But you are right setting _setValueIsAdjusting(true/false)_ is a much better solution. 
I just tested this and it works fine for me.